diff --git i/config.cfg w/config.cfg
index b0c7756..2c0cd98 100644
--- i/config.cfg
+++ w/config.cfg
@@ -6,8 +6,11 @@
 # IPsec configuration but WireGuard clients must each use a unique
 # WireGuard configuration.
 users:
-  - dan
-  - jack
+  - hush
+  - partofme
+  - phone
+  - tozt
+  - mail
 
 # NOTE: If your usernames have leading 0's, like "000dan", you have to escape them
 
@@ -31,7 +34,7 @@ wireguard_port: 51820
 # automatically based on your server, but if connections hang you might need to
 # adjust this yourself.
 # See: https://github.com/trailofbits/algo/blob/master/docs/troubleshooting.md#various-websites-appear-to-be-offline-through-the-vpn
-reduce_mtu: 0
+reduce_mtu: 184
 
 # StrongSwan log level
 # https://wiki.strongswan.org/projects/strongswan/wiki/LoggerConfiguration
@@ -92,7 +95,7 @@ VPN_PayloadIdentifier: "{{ 800000 | random | to_uuid | upper }}"
 CA_PayloadIdentifier: "{{ 700000 | random | to_uuid | upper }}"
 
 # Block traffic between connected clients
-BetweenClients_DROP: true
+BetweenClients_DROP: false
 
 congrats:
   common: |
